AllReadyApp/Web-App/AllReady/package.json

```diff
@@ -9,7 +9,9 @@
     "gulp-uglify": "1.2.0",
     "bower": "1.6.5",
     "rimraf": "2.2.8",
+    "gulp-tslint": "3.6.0",
     "gulp-typescript": "2.9.2",
```

Comment:
We should take a dependency on typescript and pass it into gulp-typescript so that we don't rely on people having typescript 1.6.X installed on their machines.